src/Form/matching.js

```diff
@@ -369,13 +399,18 @@ class Matching {
     /**
      * If we get here, a firefox/vendor regex was given and we can execute it on the element
      * strings
-     * @param {RegExp} regex
-     * @param {HTMLInputElement|HTMLSelectElement} el
-     * @param {HTMLFormElement} form
+     * @param {string} lookup
      * @return {MatchingResult}
      */
-    execVendorRegex (regex, el, form) {
-        for (let elementString of this.getElementStrings(el, form)) {
+    execVendorRegex (lookup) {
+        const regex = this.vendorRegex(lookup)
+        if (!regex) {
+            return { matched: false }
+        }
+        /** @type {MatchableStrings[]} */
+        const stringsToMatch = ['nameAttr', 'labelText', 'placeholderAttr', 'id', 'relatedText']
```

Comment:
Didn't we discuss about removing the `id`? I see it was already there, but I can't remember if we had this convo and where we landed.

Reply:
`id` is only being used on the vendor regexes now - if I remove it we get an additional 4 failures.